ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -343,7 +343,7 @@ protected Boolean findNextCompactionAndExecute(boolean collectGenericStats, bool
       findNextCompactRequest.setWorkerId(workerName);
       findNextCompactRequest.setWorkerVersion(runtimeVersion);
       ci = CompactionInfo.optionalCompactionInfoStructToInfo(msc.findNextCompact(findNextCompactRequest));
-      LOG.debug("Processing compaction request " + ci);
+      LOG.info("Processing compaction request {}", ci);
```

Comment:
i don't think it should be under info as there is more informative log entry a bit lower
````
LOG.info("Starting " + ci.type.toString() + " compaction for " + ci.getFullPartitionName() + ", id:" +
                ci.id + " in " + compactionTxn + " with compute stats set to " + computeStats);
````